src/modules/settings/components/Window.jsx

```diff
@@ -29,7 +30,7 @@ function Page(props) {
 }
 
 function isWindowSmallStandaloneChat() {
-  return window.innerWidth < 600 && window.location.pathname.endsWith('/chat');
+  return window.innerWidth < 600 && isExternalWindow();
```

Comment:
we should pass in a predicate instead, rather than making a util that works for all sites. this also isn't checking for an external window, it's checking for a standalone chat window